css-overflow-5/Overview.bs

```diff
@@ -155,6 +155,28 @@ its content will be integrated into this specification,
 which will then replace it.
 Until then, this specification only contains additions and extensions to level 4.
 
+<h2 id="overflow-concepts">
+Overflow Concepts and Terminology</h2>
+
+Issue: Copy [[css-overflow-3#overflow-concepts|Level 3 content]] when final.
+
+<h3 id="scrolling">Scrolling overflow</h3>
+
+The following is added to the concepts in scrolling overflow:
+
+Every [=scroll container=] maintains a <dfn export>current scroll target</dfn>.
+It is initially <code>null</code>,
+and is reset to <code>null</code> after any scrolling operations in that [=scroll container=]
+initiated by the user,
+or any script initiated operations that do not have a target,
+or when the target is removed from the document.
+Scrolling operations in the [=scroll container=] with a target Element or PseudoElement,
+set the [=current scroll target=] to that target Element or PseudoElement.
+
+Issue: Setting the current scroll target should be defined in how to <a>scroll a target into view</a>.
```

Comment:
Do you think I should make these edits in the other specs with this PR or leave them as issues here?

Reply:
Can leave as issues for the moment.

Reply:
Sounds good!